### ISSUE TITLE
Map Generation

### ISSUE THREAD

**Issue by lilrooness** [DanjoeGames/airships#1]
Diamond Square algorithm to generate random map (of so far undetermined size :+1: for well defined work)

Interested in using: https://github.com/danprince/tiny-binary-format

**Comment by danprince** [comment]
Yay. My libraries!

**Comment by lilrooness** [comment]
Yeah, if eventually we want to do partial map updates and stuff over websockets, I'd rather prepare for that now xD

**Comment by danprince** [comment]
That's the power of binary over websockets + typed arrays + tiny binary format. We can basically just stream the map.
